### DISCUSSION TITLE
posts/transcribe-trial-reset/

### DISCUSSION THREAD

**Discussion by giscus** [ZhymabekRoman/zhymabekroman.github.io#3] [General]
# posts/transcribe-trial-reset/

My thoughts, projects and etc.

https://zhymabekroman.github.io/posts/transcribe-trial-reset/

**Comment by atengberg** [comment]
Open source to the rescue: [SpeechNote](https://flathub.org/apps/net.mkiol.SpeechNote)

**Comment by TrantaLocked** [comment]
How to reset for Windows?

**Reply by ZhymabekRoman** [reply]
Idk. Just use Keygen.

**Comment by Merlin04** [comment]
Seems like transcribe now also will sometimes create `~/Documents/nolistEF78.rr`, yet another file that holds some junk data and also day the trial started. If deleting `.yeqwqauh` and removing the `InstallDate` key from your config file doesn't work, try also deleting the nolistEF78 file, then run `sudo touch` and `sudo chmod 400` on each of them for good measure so transcribe can't create them again

**Reply by ZhymabekRoman** [reply]
Thanks, what version of Transcribe did you use?

**Reply by Merlin04** [reply]
I'm using 9.42.0, downloaded a few months ago I think?
(also, if the devs ever happen to change the filenames or add more, you can probably find them with `strace -f -e trace=open,openat,read,write,stat,access ./transcribe 2>&1 | tee trace.log`, then open the trace file in an editor and look around the area where the other config files are opened)

**Reply by ZhymabekRoman** [reply]
okkay, thanks
